packages/twenty-front/src/modules/object-record/relation-picker/components/SingleEntitySelectMenuItemsWithSearch.tsx

```diff
@@ -104,6 +99,8 @@ export const SingleEntitySelectMenuItemsWithSearch = ({
           {results}
         </>
       )}
+      {entities.entitiesToSelect.length > 0 && <DropdownMenuSeparator />}
```

Comment:
My bad, this separator is needed before the `SearchInput` a few lines down.

Reply:
Haha sorry, this AI is often irrelevant

Reply:
Yeah... it had a point though!
I forgot to add the "Add New" button in case `dropdownPlacement` included `end`. Nothing to do with its comment, but it made me realise 😂